client/components/drop-zone/test/index.jsx

```diff
@@ -37,7 +24,9 @@ describe( 'DropZone', function() {
 	} );
 
 	after( function() {
-		delete window.MutationObserver;
+		if ( global.window && global.window.MutationObserver ) {
```

Comment:
Not related to this PR, but maybe we should improve the way clean window document in `useFakeDom` helper to avoid manual cleaning?

/cc @aduth @blowery